### PR TITLE
Update develop branch and fix announcement cell

### PR DESCRIPTION
**Pull request by paulinebessoles**
This PR aims to update develop branch with late commits from master and to fix empty announcement cell.

### CODE REVIEW

File: app/views/decidim/newsletters/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<% @cell = cell(
```
Comment (Quentinchampenois) [inline app/views/decidim/newsletters/show.html.erb:1]: Hey @paulinebessoles,

Comparing this file with [the original](https://github.com/decidim/decidim/blob/release/0.24-stable/decidim-core/app/views/decidim/newsletters/show.html.erb) I don't see significative changes, does this monkey patch is still needed ?

File: config/application.rb [hunk 2]
```diff
@@ -19,7 +19,7 @@ class Application < Rails::Application
 
     # This needs to be set for correct images URLs in emails
     # DON'T FORGET to ALSO set this in `config/initializers/carrierwave.rb`
-    # config.action_mailer.asset_host = "https://participer.loire-atlantique.fr"
+    config.action_mailer.asset_host = "https://participer.loire-atlantique.fr"
```
Comment (Quentinchampenois) [inline config/application.rb:22]: To ensure homogeneity between `config/initializers/carrierwave.rb` and this file we should add a guard clause like following: 

```suggestion
    config.action_mailer.asset_host = "https://participer.loire-atlantique.fr" if Rails.env.production?
```

File: config/initializers/decidim.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@
 
   # Change these lines to set your preferred locales
   config.default_locale = :fr
-  config.available_locales = [:fr]
+  config.available_locales = [:fr, :en]
```
Comment (Quentinchampenois) [inline config/initializers/decidim.rb:11]: Does client wants english language on the platform ? If yes, it is good for me

**Review by Quentinchampenois — CHANGES_REQUESTED** [review]
Thanks for your work ! 
Tests seems to be broken for now

File: app/views/decidim/newsletters/show.html.erb (hunk 1 above)
Comment (paulinebessoles) [reply app/views/decidim/newsletters/show.html.erb:1]: You're right, I delete it!

**Review by Quentinchampenois — APPROVED** [review]
Awesome, seems good to me !

**Review by armandfardeau — APPROVED** [review]

**Pull request merged**